Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -216,6 +225,23 @@ theorem map_comp_map {I : Type*} [Group I] (M : Subgroup H) (O : Subgroup I) [M.
     (map M O g hg).comp (map N M f hf) = map N O (g.comp f) hgf :=
   MonoidHom.ext (map_map N M O f g hf hg hgf)
 
+@[to_additive (attr := simp)]
+theorem comap_map_mk' (N H : Subgroup G) [N.Normal] :
+    Subgroup.comap (mk' N) (Subgroup.map (mk' N) H) = N ⊔ H :=
+  by simp [Subgroup.comap_map_eq, sup_comm]
+
+/-- The **correspondence theorem**, or lattice theorem,
+  or fourth isomorphism theorem for multiplicative groups-/
+@[to_additive "The **correspondence theorem**, or lattice theorem,
+  or fourth isomorphism theorem for additive groups"]
+def comap_mk'_OrderIso (N : Subgroup G) [hn : N.Normal] :
```

Comment:
The naming convention says that all definitions should be `camelCase`:
```suggestion
def comapMk'OrderIso (N : Subgroup G) [hn : N.Normal] :
```

Reply:
Suggestion adopted, thanks!